app/models/picture.rb

```diff
@@ -5,10 +5,11 @@ class Picture
   embedded_in :photographic, inverse_of: :pictures, polymorphic: true
 
   has_mongoid_attached_file :attachment,
-    styles: { small:  ['100x100#', :jpg],
+    styles: { small:  ['100x100^', :jpg],
```

Comment:
lesser of two evils I think. Indenting these means indenting convert_options, and then what?